src/MizarWidgets/MizarMainWindow.cpp

```diff
@@ -12,6 +12,10 @@ namespace orbit_mizar_widgets {
 
 MizarMainWindow::MizarMainWindow() : ui_(std::make_unique<Ui::MainWindow>()) { ui_->setupUi(this); }
 
+void MizarMainWindow::Init(const orbit_mizar_data::BaselineAndComparison* baseline_and_comparison) {
+  ui_->sampling_with_frame_track_widget_->Init(baseline_and_comparison);
+}
```

Comment:
Why pointer instead of const ref? Here and in `SamplingWithFrameTrackWidget::Init`

Reply:
I've spent some time contemplating the options. Pass by pointer emphasises that the ownership is not transferred at the call cite.

Reply:
Would it be an option to make `baseline_and_comparison` a constructor argument? This would make it clearer that to the user of the class that the referred object needs to be kept alive until the main window gets destroyed. Having this in a function makes this a lot harder.

Reply:
@antonrohr it's fine for now. Soon it won't be.